frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -338,13 +338,7 @@ const ManageHostsPage = ({
 
   useQuery<IPolicyAPIResponse, Error>(
     ["policy"],
-    () => {
-      const teamId = parseInt(queryParams?.team_id, 10) || 0;
-      const request = teamId
-        ? teamPoliciesAPI.load(teamId, policyId)
-        : globalPoliciesAPI.load(policyId);
-      return request;
-    },
+    () => globalPoliciesAPI.load(policyId),
```

Comment:
Is there a reason why only the `globalPoliciesAPI` is being used?

Reply:
There is a new requirement that we keep policy filters across team changes. The issues was that I was on Team A with Policy 1 and switched to Team B with Policy 1 still applied, and than refreshed the page, I need to get the information for Policy 1 still, so the call to `teams/2/policies/1` will not work in this case. We have to look through all policies that exist with the endpoint `/policies/1`

Reply:
Gotcha. IIRC each policy has a unique id so that makes sense. I don't recall if there is anything tricky with the RBAC permissions on the different policies endpoints. It would probably be worth double checking that team admins, team maintainers, and team observers can all access that global endpoint in the way you are expecting. And also what happens when a team user tries to use a team policy id for a team that they don't belong too.

Reply:
currently is seems to return the team response. It's a different question if there should be some permissions around this. I checked with both global team and individual team roles and they all seem to have access to `/latest/policies/:id` endpoint